contrib/ruby/lib/trilogy.rb

```diff
@@ -1,6 +1,8 @@
 require "trilogy/version"
 
 class Trilogy
+  attr_reader :connection_options
```

Comment:
Do think it's worth dup-ing and freezing on read to limit folks mucking around with the original hash passed in?

```suggestion
  def connection_options
    @connection_options.dup.freeze
  end
```

I suppose folks could still muck around with the hash values, but that seems somewhat less likely.

Reply:
Makes sense to me 👍